### DISCUSSION TITLE
Commercial use of supervision

### DISCUSSION THREAD

**Discussion by NicolasGonzalezF** [roboflow/supervision#1564] [Q&A] [answered]
Just a small quick question, 1. can i use supervision for commercial porpuses in a way that doesnt force me to have my code be open source , 2. if i train on a custom dataset is viable to use in commercial porpuses? i understand supervision uses some ultralytics tools so idk if its allowed

**Comment by onuralpszr** [comment]
@NicolasGonzalezF   For supervision yes you can use in commercial and  license MIT and you checkout from here as well.(https://github.com/roboflow/supervision/blob/develop/LICENSE.md) and for ultralytics please check their licenses because they have some commercial uses you need to check as well. 

Also we don't use  ANY "ultralytics" tool from their repo we don't even install with supervision

**Reply by NicolasGonzalezF** [reply]
![image](https://github.com/user-attachments/assets/a79ad5f3-0cba-4a02-b0f9-d8046569641c)
but you guys use ultralytics for object detection , which they have another license that doesnt affect de code?
or is the inferance method just equally good ?
![image](https://github.com/user-attachments/assets/905550a2-70d1-4dea-93dd-5901da68c3f5)
 sorry for the question im a bit loss since the discord chat is a bit dead

**Reply by LinasKo** [reply]
The performance of the same model on inference should be the same. There are restrictions on using Ultralytics commercially, but I'm not entirely familiar with their terms. Roboflow models can be used for commercial purposes if you have the Starter plan. https://roboflow.com/pricing

**Reply by NicolasGonzalezF** [reply]
> s. Roboflow models can be used for commercial purposes if you have the Starte

so i can use roboflow's supervision for object detection without needing to be open source, and can train with supervision or well detectron my custom dataset and keep its rights? cuz thats what puts me off from Utralytics that the trained model is their property

**Reply by LinasKo** [reply, marked as answer]
You may use supervision freely - it's completely open-source, under MIT license.

In terms of licensing, I have to admit a lack of knowledge myself.
This page could help, perhaps? https://roboflow.com/licensing
